Mathlib/MeasureTheory/Constructions/Polish.lean

```diff
@@ -842,37 +946,65 @@ theorem measurableSet_exists_tendsto [hγ : OpensMeasurableSpace γ] [Countable
 
 end MeasureTheory
 
+namespace StandardBorelSpace
+
+variable {α : Type*} [MeasurableSpace α] [StandardBorelSpace α]
+
+/-- If `s` is a measurable set in a standard Borel space, there is a compatible Polish topology
+making `s` clopen. -/
+theorem _root_.MeasurableSet.isClopenable' {s : Set α} (hs : MeasurableSet s) :
+    ∃ _ : TopologicalSpace α, BorelSpace α ∧ PolishSpace α ∧ IsClosed s ∧ IsOpen s := by
+  letI := upgradeStandardBorel α
+  obtain ⟨t, hle, ht, s_clopen⟩ := hs.isClopenable
+  refine' ⟨t, _, ht, s_clopen⟩
+  constructor
+  rw [eq_borel_upgradeStandardBorel α, borel_eq_borel_of_le ht _ hle]
+  infer_instance
+
+/-- A measurable subspace of a standard Borel space is standard Borel. -/
+theorem _root_.MeasurableSet.standardBorel {s : Set α} (hs : MeasurableSet s) :
+    StandardBorelSpace s := by
+  obtain ⟨_, _, _, s_closed, _⟩ := hs.isClopenable'
+  haveI := s_closed.polishSpace
+  infer_instance
+
+end StandardBorelSpace
+
 /-! ### The Borel Isomorphism Theorem -/
 
 namespace PolishSpace
 
-variable {β : Type*} [TopologicalSpace β] [PolishSpace α] [PolishSpace β]
+variable {α β : Type*}
```

Comment:
Same thing: don't introduce a new type with the same name as a type which is already in scope.

Reply:
should all be fixed now.